components/com_content/tmpl/form/edit.php

```diff
@@ -48,106 +47,97 @@
+                    <?php echo $this->form->renderField('urlatext', 'urls'); ?>
+                    <div class="control-group">
+                        <div class="controls">
+                            <?php echo $this->form->getInput('targeta', 'urls'); ?>
+                        </div>
+                    </div>
+                    <?php echo $this->form->renderField('urlb', 'urls'); ?>
+                    <?php echo $this->form->renderField('urlbtext', 'urls'); ?>
                     <div class="control-group">
-                        <div class="control-label">
+                        <div class="controls">
+                            <?php echo $this->form->getInput('targetb', 'urls'); ?>
                         </div>
+                    </div>
+                    <?php echo $this->form->renderField('urlc', 'urls'); ?>
+                    <?php echo $this->form->renderField('urlctext', 'urls'); ?>
+                    <div class="control-group">
                         <div class="controls">
-                            <?php echo Text::_('COM_CONTENT_ORDERING'); ?>
+                            <?php echo $this->form->getInput('targetc', 'urls'); ?>
                         </div>
                     </div>
+                    <?php echo HTMLHelper::_('uitab.endTab'); ?>
                 <?php endif; ?>
-            <?php echo HTMLHelper::_('uitab.endTab'); ?>
 
-            <?php if (Multilanguage::isEnabled()) : ?>
-                <?php echo HTMLHelper::_('uitab.addTab', $this->tab_name, 'language', Text::_('JFIELD_LANGUAGE_LABEL')); ?>
+                <?php echo LayoutHelper::render('joomla.edit.params', $this); ?>
+
+                <?php echo HTMLHelper::_('uitab.addTab', $this->tab_name, 'options', Text::_('JOPTIONS')); ?>
+                    <?php echo $this->form->renderField('transition'); ?>
+                    <?php echo $this->form->renderField('state'); ?>
+                    <?php echo $this->form->renderField('catid'); ?>
+                    <?php if ($this->item->params->get('access-change')) : ?>
+                        <?php echo $this->form->renderField('featured'); ?>
+                    <?php endif; ?>
+                    
+                    <?php echo $this->form->renderField('tags'); ?>
+                    <?php echo $this->form->renderField('note'); ?>
+                    <?php if ($params->get('save_history', 0)) : ?>
+                        <?php echo $this->form->renderField('version_note'); ?>
+                    <?php endif; ?>
+                    <?php echo $this->form->renderField('access'); ?>
+                    <?php if (is_null($this->item->id)) : ?>
+                        <div class="control-group">
+                            <div class="control-label">
+                            </div>
+                            <div class="controls">
+                                <?php echo Text::_('COM_CONTENT_ORDERING'); ?>
+                            </div>
```

Comment:
```suggestion
                            <div class="control-label">
                            </div>
                            <div class="controls">
                                <?php echo Text::_('COM_CONTENT_ORDERING'); ?>
                            </div>
```
Please check. Empty div and string in controls div?

Reply:
thx, this is completely wrong, but not quit in scope of this PR.  
It displays a description and . 

![grafik](https://github.com/user-attachments/assets/5c4de33f-0043-4a73-8a40-05d14a934b60)

My impulse is to remove the whole field from frontend editing. What's your opinion?

Reply:
empty div removed